tests/test-requirements.txt

```diff
@@ -0,0 +1,5 @@
+pytest
+pytest-timeout ~= 2.3.1
```

Comment:
Don't think we need this constraint.

Reply:
done in [remove test-requrements.txt, moved to project.toml](https://github.com/CQCL/quantinuum-qircheck/pull/1/commits/e0cdc89c0f3ac225b500b6d12e370759b9da165e)
[e0cdc89](https://github.com/CQCL/quantinuum-qircheck/pull/1/commits/e0cdc89c0f3ac225b500b6d12e370759b9da165e)

and 
[remove test-requrements.txt, moved to project.toml II](https://github.com/CQCL/quantinuum-qircheck/pull/1/commits/d1912a6100cb03c0d37c7f8aeadda702af33b73f)
[d1912a6](https://github.com/CQCL/quantinuum-qircheck/pull/1/commits/d1912a6100cb03c0d37c7f8aeadda702af33b73f)